bigchaindb/common/schema/transaction.yaml

```diff
@@ -131,21 +130,27 @@ definitions:
         - type: object
           additionalProperties: true
         - type: 'null'
-  condition:
+  output:
     type: object
     description: |
-        An output of a transaction. A condition describes a quantity of an asset
-        and what conditions must be met in order for it to be fulfilled. See also:
-        fulfillment_.
+        A transaction output. Describes a quantity of an asset and a locking
```

Comment:
The "locking script" terminology isn't used anywhere else in our docs, as far as I know. We might use it in the future, but for now I think we can do without. How about:

A transaction output. Describes the quantity of an asset and the requirements that must be met to spend the output.